### DISCUSSION TITLE
(mini.pick) how to use picker for input and select?

### DISCUSSION THREAD

**Discussion by mainendra** [nvim-mini/mini.nvim#645] [Q&A] [answered]
is there any easy way to use `mini.pick` for `vim.ui.input` and `vim.ui.select`? similar to `stevearc/dressing.nvim`
Actually I want to use it for `code_action` list.

**Comment by echasnovski** [comment, marked as answer]
`vim.ui.select` - [yes](https://github.com/echasnovski/mini.nvim/blob/e578f353d0b1d9f21a9640458d8e506c0c9acf5e/doc/mini-pick.txt#L27).
`vim.ui.input` - no, as this doesn't involve picking from an array.

Please, read help page.
